sql/moz-fx-data-shared-prod/mozilla_vpn_derived/users_attribution_v1/query.sql

```diff
@@ -0,0 +1,7 @@
+SELECT
+  CAST(flow_id AS INTEGER) AS user_id,
+  attribution
+FROM
+  `moz-fx-data-shared-prod.mozilla_vpn_derived.fxa_attribution_v1`
```

Comment:
@relud I just noticed a `mozilla_vpn_external.users_attribution_v1` table that was created in May 2022, which I'm guessing was your doing.  It seems to have more Guardian attributions (621,771) than were added to `mozilla_vpn_derived.fxa_attribution_v1` (489,574).

Do you think I should use `mozilla_vpn_external.users_attribution_v1` as a data source for this instead?

Or was there a reason the smaller subset of attributions was added to `mozilla_vpn_derived.fxa_attribution_v1`?

Reply:
> @relud I just noticed a `mozilla_vpn_external.users_attribution_v1` table that was created in May 2022, which I'm guessing was your doing.

seems likely

> It seems to have more Guardian attributions (621,771) than were added to `mozilla_vpn_derived.fxa_attribution_v1` (489,574).
> 
> Do you think I should use `mozilla_vpn_external.users_attribution_v1` as a data source for this instead?

if the old table looks the same as this new table, but with more entries, then I think we should use the old table. if the attribution is different, let's use the new table.

> Or was there a reason the smaller subset of attributions was added to `mozilla_vpn_derived.fxa_attribution_v1`?

no idea what it would be a smaller subset

Reply:
`mozilla_vpn_derived.fxa_attribution_v1` only has Guardian attributions up until 2021-05-31, while `mozilla_vpn_external.users_attribution_v1` has those same attributions (no differences in the attribution values) plus more between 2021-06-01 and 2022-03-23.

Since VPN's SubPlat funnel wasn't deployed until 2021-08-25 I think it's worth switching to use `mozilla_vpn_external.users_attribution_v1` to get the additional attributions between 2021-06-01 and 2021-08-25 (I'm going to have the SubPlat ETL ignore any Guardian attributions after 2021-08-25, since those should be in the FxA logs).

Reply:
sounds good